presto-spi/src/main/java/io/prestosql/spi/security/AccessDeniedException.java

```diff
@@ -156,6 +156,11 @@ public static void denyShowColumnsMetadata(String tableName)
         throw new AccessDeniedException(format("Cannot show columns of table %s", tableName));
     }
 
+    public static void denyShowTableStats(String tableName)
```

Comment:
Since this isn't used (or needed) by the SPI. I would inline this into the one call site in the main module.

Reply:
Done.